### PR TITLE
wallet reskin

### PR DESCRIPTION
**Pull request by manuel-calavera**

### CODE REVIEW

**Comment by CriesofCarrots** [comment]
>     * It would be nice to have some kind of responsiveness when the icon buttons are clicked, for instance, when I copy an address, there's no indication that anything happened

Whatever you come up with for this, let's backport to the network explorer :)

**Comment by manuel-calavera** [comment]
@jstarry i will take care of 1 and 2 tmrw. 4 is up to you and 3 and 5 could you please check with designers on it. i need design before to make it happen.

**Comment by manuel-calavera** [comment]
@jstarry please check one more time and let's merge it. we can do other changes later as soon as we have design for it.

File: src/account.js [hunk 1]
```diff
@@ -148,14 +151,21 @@ export class Account extends React.Component {
...
+          </HelpBlock>
         </FormGroup>
-        <hr />
-        <Button bsStyle="primary" onClick={() => this.recoverAccount()}>
-          Recover Account
-        </Button>
+        <div className="text-center-xs">
+          <Button
+            disabled={!this.state.recoveredPhrase}
+            bsStyle="primary"
```
Comment (jstarry) [inline src/account.js]: This is leftover right?

File: src/settings.js [hunk 2]
```diff
@@ -92,52 +93,50 @@ export class Settings extends React.Component {
...
+          <HelpBlock>{this.state.validationHelpBlock}</HelpBlock>
+        </FormGroup>
+        <div className="mt40">
+          <h5>Account Settings</h5>
+          <p className="text">
+            <span className="green">WARNING:</span>
+            &nbsp;Any tokens associated with the current account will be lost
+          </p>
+          <div className="text-center-xs">
             <Button bsStyle="danger" onClick={() => this.resetAccount()}>
```
Comment (jstarry) [inline src/settings.js]: This `bsStyle` prop should be removed

File: src/wallet.js [hunk 3]
```diff
@@ -549,56 +569,81 @@ export class Wallet extends React.Component {
...
+              <Well>
+                <h4>
+                  Account Balance
+                </h4>
+                <div className="balance">
+                  <div className="balance-val">
+                    {this.state.balance}
+                  </div>
+                  <div className="balance-ttl">
+                    Sols
```
Comment (jstarry) [inline src/wallet.js]: Please update to Lamports

**Comment by jstarry** [comment]
@manuel-calavera also the popup window definitely regressed with this change, could you maybe just adjust the margins until we get a proper design? Here's what it looks like with your changes:

Screenshot 2019-08-16 14 20 12

File: src/wallet.js (hunk 3 above)
Comment (CriesofCarrots) [reply src/wallet.js]: I think we want to move toward using SOLs consistently across our user-facing products. But presumably might need a little backend updating.

**Comment by manuel-calavera** [comment]
@jstarry done ;)

**Pull request merged**